src/transport/manager/mod.rs

```diff
@@ -233,7 +233,7 @@ pub struct TransportManager {
     peers: Arc<RwLock<HashMap<PeerId, PeerContext>>>,
 
     /// Handle to [`crate::transport::manager::TransportManager`].
-    transport_manager_handle: TransportManagerHandle,
+    pub transport_manager_handle: TransportManagerHandle,
```

Comment:
Could we introduce a new wrapper method for this one instead? 🤔 

I thought we already have one, but that's not the case 🤔 

```rust
    /// Acquire the `transport_manager_handle`.
    pub fn transport_manager_handle(&self) -> TransportManagerHandle {
       self.transport_manager_handle
    }
```